crates/node_binding/src/fs_node/write.rs

```diff
@@ -165,12 +166,12 @@ impl ReadableFileSystem for NodeFileSystem {
       .to_fs_result()
       // TODO: simplify the return value?
       .map(|result| match result {
-        Either3::A(buf) => buf.into(),
-        Either3::B(str) => str.into(),
-        Either3::C(_) => vec![],
+        Either3::A(buf) => Bytes::from_owner(buf),
+        Either3::B(str) => Bytes::from_owner(str),
```

Comment:
If it is a `Vec`, here should use `Bytes::from`. this has less overhead than `from_owner`.